### ISSUE TITLE
won't install on osx

### ISSUE THREAD

**Issue by eydrian** [uzh/gc3pie#539]
`env CC=gcc ./setup.py install` failes with 

```
build/temp.macosx-10.11-intel-2.7/_openssl.c:423:10: fatal error:
      'openssl/opensslv.h' file not found
#include <openssl/opensslv.h>
```

The following workaround is necessary to successfully install gc3pie

```
brew install openssl
env LDFLAGS="-L$(brew --prefix openssl)/lib" CFLAGS="-I$(brew --prefix openssl)/include" pip install cryptography
env CC=gcc ./setup.py install
```

It might be related to the latest OS X update => 10.11.5

**Comment by arcimboldo** [comment]
Yes, apple stopped releasing openssl libraries (and therefore headers)

http://rentzsch.tumblr.com/post/33696323211/wherein-i-write-apples-technote-about-openssl-on
https://lists.apple.com/archives/macnetworkprog/2015/Jun/msg00025.html

Not sure why this is not a problem on Linux (or maybe it is and I don't know it)

The fix worked for me though on MacOSX 10.11.5  (El Capitan)
